defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+log_level: DEBUG
+
```

Comment:
match the monasca_log_level from the other roles, should have the same name. Also we default this to warn in our other roles